participants/Everyone's ipfslife memories.md

```diff
@@ -0,0 +1,52 @@
+Before submitting a PR containing your team's project information:
+•	Make sure you rename this file to contain your project's name
+•	Respond to add your response in each section below
+
+(0) Registration Submission, Project Name, and Team Slack Handles
+Ans: (Submission ID: 670281357), 
+Project Name: Everyone's ipfslife memories 
+Team Slack Handles: https://app.slack.com/client/TEHTVS1L6/D01BPRQKXB4/user_profile/U01CDM3JBL0
+
+(1) Brief Project Description
+Ans: Store Everyone's ipfslife memories, Never lose.
+
+(2) Link to Project UI
+Ans: http://spacerace.ipfslife.io:38080/sr2 , And code in https://github.com/leeguanjun/sr2
+
+(3) What does your application/UI do?
+Ans: Through the distributed storage application, it provides the complete function operation of the InterPlanetary effect.
+
+(4) If your project is using a curated dataset, which one are you using?
+Ans: Mapping NoSQL and Data CID in index. No 
+
+(5) If your project is not using a curated dataset, please tell us a little bit more about your data by answering the questions below.
+Ans: life video,photo,voice,diary. No, No, No  . 
```

Comment:
Do you have the rights to use this data? And how do you ensure that copyrighted materials are not uploaded?

Reply:
Hi cwaring
We have learned that these videos are mainly about local scenery, which are all open to the public. Now they are only test data for this slingshot competition. We do not do commercial profit-making use. It will not have any impact.

In the future, it will only be provided to individual users to save their precious personal file materials.

Reply:
Ok, great thanks for confirming

Reply:
Hi cwaring 
Thank you very much for your reply soon.